mitmproxy/addons/__init__.py

```diff
@@ -20,6 +20,7 @@
 from mitmproxy.addons import streambodies
 from mitmproxy.addons import save
 from mitmproxy.addons import upstream_auth
+from mitmproxy.addons import upload
```

Comment:
Let's call this `share` and not `upload` - the user wants to share their flows, uploading is just the implementation of that. :)

Reply:
Okay